### DISCUSSION TITLE
How to add different color scheme to different panes?

### DISCUSSION THREAD

**Discussion by muldri** [Eugeny/tabby#8773] [Q&A]
Hello
My work requires to be connected to two different hosts and run different commands iteratively.  I find having two panes in the same tab to do that.  However all my split panes share the same color scheme.  It would be ideal to have the ability to change the color scheme for different panes to keep track of which pane is connected to which host.

Is there a way to do this in tabby ?   Thanks in advance for all the help.
Murali

**Comment by Eugeny** [comment]
You can only have a different color scheme with a different profile. Create two profiles with different schemes, open them in two tabs, then drag the other tab onto the side of the active tab to combine them into split panes. You can then right-click the combined tab and "save as a layout" if you wish

**Comment by muldri** [comment]
Hello
Thank you very much for the reply. Really appreciate it.  Let me try that. 
Once i open multiple panes it is not intuitive to close them. Right now i dug through hotkeys and set one to close the pane.  An x icon on top left of pane may make that task easier.   
Hope this would be considered.    Keep up the great work
Murali


Sent from Yahoo Mail for iPhone


On Tuesday, August 1, 2023, 1:57 PM, Eugene ***@***.***> wrote:




You can only have a different color scheme with a different profile. Create two profiles with different schemes, open them in two tabs, then drag the other tab onto the side of the active tab to combine them into split panes. You can then right-click the combined tab and "save as a layout" if you wish
